TEI.js

```diff
@@ -1,140 +1,688 @@
+	jurisdiction: "jurisdiction", // not zot
+	language: "language",
+	license: "rights",
+	locator: "locator", // not zot
+	medium: "medium", // zot:artworkMedium, zot:audioRecordingFormat, zot:system, zot:videoRecordingFormat, zot:interviewMedium, zot:audioFileType
+	note: "extra",
+	number: "number", // zot:billNumber
+	"number-of-pages": "numPages",
+	"number-of-volumes": "numberOfVolumes",
+	"original-date": "original-date", // not zot
+	"original-publisher": "original-publisher", // not zot
+	"original-publisher-place": "original-publisher-place", // not zot
+	"original-title": "original-title", // not zot
+	page: "page", // zot:page, zot:codePage
+	// "page-first": "page-first", // not zot
+	"part-number": "part-number", // not zot
+	"part-title": "part-title", // not zot
+	pmcid: "PMCID",
+	pmid: "PMID",
+	publisher: "publisher",
+	"publisher-place": "place",
+	references: "references", // zot:history, zot:references
+	"reviewed-genre": "reviewed-genre", // not zot
+	"reviewed-title": "reviewed-title", // not zot
+	scale: "scale",
+	section: "section", // zot:section, zot:committee
+	source: "libraryCatalog",
+	status: "legalStatus",
+	submitted: "filingDate",
+	"supplement-number": "supplement-number", // not zot
+	title: "title",
+	"title-short": "shortTitle",
+	url: "URL",
+	version: "versionNumber",
+	volume: "volume",
+	"year-suffix": "year-suffix", // not zot
+};
+
+/**
+ * For a parser of zot:extra field,
+ * List of CSL properties with zotero key if available.
+ * These properties are repeatable and are appended.
+ * References https://aurimasv.github.io/z2csl/typeMap.xml
+ */
+const cslCreators = {
+	author: "author",
+	chair: "chair", // not zot
+	"collection-editor": "seriesEditor",
+	compiler: "compiler", // not zot
+	composer: "composer",
+	"container-author": "container-author",
+	contributor: "contributor",
+	curator: "curator", // not zot
+	director: "author", // cinema
+	editor: "editor",
+	"editor-translator": "editorial-translator", // not zot
+	"editorial-director": "editorial-director", // not zot
+	"executive-producer": "executive-producer", // not zot
+	guest: "guest",
+	host: "host", // not zot
+	illustrator: "illustrator", // not zot
+	interviewer: "interviewer",
+	narrator: "narrator", // not zot
+	organizer: "organizer", // not zot
+	"original-author": "original-author", // not zot
+	performer: "performer",
+	recipient: "recipient",
+	"reviewed-author": "reviewedAuthor",
+	"script-writer": "scriptWriter",
+	translator: "translator",
+};
+
+
+/**
+ * Parse the “extra” field to get “extra fields”.
+ * Adaptation of Zotero.Utilities.Item.extraToCSL()
+ * https://github.com/zotero/utilities/blob/9c89b23153ce621ed0f1d581a5e32248704c6fb7/utilities_item.js#L614
+ * Syntax, one property by line
+ *
+ * known-propertie: value
+ * Othe Known: value
+ *
+ *
+ * This: could be a free note.
+ * [2024-01 FG]
+ * @param {*} item
+ * @returns
+ */
+function parseExtraFields(item) {
```

Comment:
I don’t understand your question, probably because I miss something about Zotero. In the context of a translator, I know how to access to Extra field content, but haven’t found another way than parse it to get extra CSL fields. Those fields may have important value (example upper, the reviewed-title).